### PR TITLE
The task_DEV-3 project files were added.

### PR DESCRIPTION
**Pull request by MariaEroshevich**

### CODE REVIEW

File: task_DEV-3/EntryPoint.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace task_DEV_3
+{
+  class EntryPoint
+  {
+    static void Main(string[] args)
+    {
+      try
+      {
```
Comment (IgorKheidorov) [inline task_DEV-3/EntryPoint.cs:10]: Check for number of input arguments

File: task_DEV-3/NumberConverter.cs [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      this.notation = notation;
+      CheckInputFormat();
+    }
+
+    /// <summary>
+    /// This method converts entered decimal number to the number from the new number system.
+    /// </summary>
+    /// <returns>Returns the reverse string with converted number.</returns>
+    public string ConvertDecimalNumber()
+    {
```
Comment (IgorKheidorov) [inline task_DEV-3/NumberConverter.cs:28]: What about decimalNumber =0?

File: task_DEV-3/NumberConverter.cs [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+      while (decimalNumber > 0)
+      {
```
Comment (IgorKheidorov) [inline task_DEV-3/NumberConverter.cs]: Good, very compact

File: task_DEV-3/NumberConverter.cs [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    
+    private string ReverseConvertedString()
+    {
+      char[] convertedNumberArray = convertedNumber.ToString().ToCharArray();
+      Array.Reverse(convertedNumberArray);
+      return new string(convertedNumberArray);
+    }
+
+    private void CheckInputFormat()
+    {
```
Comment (IgorKheidorov) [inline task_DEV-3/NumberConverter.cs:49]: Why decimalNumber = 0 is exception? It is normal number

**Review by IgorKheidorov — CHANGES_REQUESTED** [review]

File: task_DEV-3/EntryPoint.cs (hunk 1 above)
Comment (AlexanderSoroka) [reply task_DEV-3/EntryPoint.cs:10]: Actually it's useless check - see chat for my point

**Review by AlexanderSoroka — APPROVED** [review]

**Pull request merged**